requirements_dev.txt

```diff
@@ -31,6 +31,7 @@ jax[cpu]; python_version < '3.10'
 fastcore; python_version < '3.10'
 pyarrow; python_version < '3.10'
 metaflow>=2.3.5; python_version < '3.10'
+transformers; python_version < '3.10'
```

Comment:
Why is this needed?  i dont think we use transformers in pytest env.. (i think we were mocking the module only)

Reply:
Yea, my bad - I screwed up indentation 🙈 
fixed here: https://github.com/wandb/client/pull/3083/commits/c939dcbb32391b58e710df6505f07631638eb61e